conans/assets/templates/new_v2_bazel.py

```diff
@@ -180,12 +198,23 @@ def test(self):
 """
 
 
+def _get_bazel_build(name):
+    _bazel_build_template = ""
+    if platform.system() == "Windows":
```

Comment:
maybe use `settings.os`/`settings_build.os`/`settings_host.os` here instead?

Reply:
I don't think so. I'm not taking into account your profile, it's only important where you ran the `conan new ` to use the correct extension. In any case, if the extension could be wrong, you can always modify it whenever you want.